src/components/charts/Vis3.js

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from 'react'
+import { Scatter } from 'react-chartjs-2'
+import DATA from './annualMeanCO2.json'
+import DATA2 from './monthlyMeanCO2.json'
+import zoomPlugin from 'chartjs-plugin-zoom'
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+} from 'chart.js'
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+  zoomPlugin
+)
+
+export const options = {
+  type: 'line',
+  responsive: true,
+  parsing: {
+    xAxisKey: 'decimal date',
```

Comment:
Check how i did with the date, could work for this also, but if my approach will work with charts that are before present